### ISSUE TITLE
Question: How to build the project from source?

### ISSUE THREAD

**Issue by frankiee12a9** [TooTallNate/Java-WebSocket#1310]
Hello hosts,

I'm really curious about the instruction on building the project from source. but i couldn't find any of them.
I've tried to run `mvn clean install` from the root of the project, where the `pom.xml` file located but got the following error:
> [ERROR] Failed to execute goal biz.aQute.bnd:bnd-maven-plugin:4.3.1:bnd-process (default) on project Java-WebSocket: bnd error: null: ConcurrentModificationException

My local environment is:

**My local machine**
```
OS name: "windows 11", version: "10.0", arch: "amd64", family: "windows"
```

**My Java version**
```
java version "17.0.3.1" 2022-04-22 LTS
Java(TM) SE Runtime Environment (build 17.0.3.1+2-LTS-6)
Java HotSpot(TM) 64-Bit Server VM (build 17.0.3.1+2-LTS-6, mixed mode, sharing)
```
*Maven version*
```
Apache Maven 3.8.6 (84538c9988a25aec085021c365c560670ad80f63)
Maven home: C:\Program Files\apache-maven-3.8.6-bin\apache-maven-3.8.6
Java version: 17.0.3.1, vendor: Oracle Corporation, runtime: C:\Program Files\Java\jdk-17.0.3.1
Default locale: en_US, platform encoding: MS949
```

**Comment by PhilipRoman** [comment]
Yes, I can reproduce this issue. Didn't notice it faster because I usually use Gradle for everything :P
Looks like we need to update the bnd plugin to whatever version fixes https://github.com/bndtools/bnd/issues/3903

**Comment by PhilipRoman** [comment]
There is an open pull request #1309 that fixes this but I haven't had the time to review it properly.
As a temporary fix you can change the version number in `pom.xml`:
```xml
<bnd.maven.plugin.version>4.3.1</bnd.maven.plugin.version>
```
Any version number above 5.1.0 should work

**Comment by PhilipRoman** [comment]
Fixed in #1309
